rules/defluent/src/NodeAnalyzer/FluentChainMethodCallNodeAnalyzer.php

```diff
@@ -88,6 +85,15 @@ public function isFluentClassMethodOfMethodCall(MethodCall $methodCall): bool
         return true;
     }
 
+    private function isCall(Expr $expr): bool
+    {
+        if ($expr instanceof FuncCall) {
```

Comment:
There is actually `MethodCall` above, not `FuncCall`

Reply:
fixed.